src/components/DailyStatistics/TitleStatistic.tsx

```diff
@@ -61,15 +62,21 @@ export const TitleStatisticComponent: FunctionComponent<TitleStatisticComponent>
   onPressPrevDay,
   onPressNextDay,
 }) => {
+  const { i18nt } = useTranslate();
+  const formatTime =
+    lastTransactionTime !== null
+      ? { distributionTime: format(lastTransactionTime, "h:mma") }
+      : { distributionTime: "-" };
+
```

Comment:
Since they're both using the same key, we can move the conditional in:

```suggestion
  const formattedLastTransactionTime =
      { lastTransactionTime: lastTransactionTime !== null ? format(lastTransactionTime, "h:mma") : "-" }

```